data/i18n/vi/vi.toml

```
@@ -124,74 +335,237 @@ other = "Khám phá cộng đồng"
+# thì của văn bản miễn là ý nghĩa là rõ ràng.
+[release_info_eol]
+other = "**%s** bắt đầu chế độ bảo trì vào ngày **%s** và Kết thúc Vòng đời vào ngày **%s**."
+
+[release_note]
+other = "Ghi chú"
+
+[release_schedule]
+other = "Lịch Trình"
+
+[release_target_date]
+other = "Ngày Đích"
+
+[release_changelog]
+other = "Changelog"
+
+[seealso_heading]
+other = "Xem Thêm"
 
 [subscribe_button]
-other = "Đăng ký"
+other = "Đăng Ký"
+
+[synopsis_heading]
+other = "Tóm Tắt"
+
+[thirdparty_message]
+other = """Phần này liên kết đến các dự án bên thứ ba cung cấp chức năng được yêu cầu bởi Kubernetes. Tác giả dự án Kubernetes không chịu trách nhiệm về những dự án này, được liệt kê theo thứ tự chữ cái. Để thêm một dự án vào danh sách này, đọc <a href="/docs/contribute/style/content-guide/#third-party-content">hướng dẫn nội dung</a> trước khi gửi thay đổi. <a href="#third-party-content-disclaimer">Thêm thông tin.</a>"""
+
+[thirdparty_message_edit_disclaimer]
+other="""Lời khuyên về nội dung của bên thứ ba"""
+
+
+[thirdparty_message_single_item]
+other = """&#128711; Mục này liên kết đến một dự án hoặc sản phẩm bên thứ ba không phải là một phần của Kubernetes chính. <a class="alert-more-info" href="#third-party-content-disclaimer">Thông tin thêm</a>"""
+[thirdparty_message_disclaimer]
+other = """<p>Các mục trên trang này đề cập đến các sản phẩm hoặc dự án bên thứ ba cung cấp chức năng được yêu cầu bởi Kubernetes. Tác giả dự án Kubernetes không chịu trách nhiệm đối với những sản phẩm hoặc dự án của bên thứ ba đó. Xem <a href="https://github.com/cncf/foundation/blob/master/website-guidelines.md" target="_blank">hướng dẫn trang web CNCF</a> để biết thêm chi tiết.</p><p>Bạn nên đọc <a href="/docs/contribute/style/content-guide/#third-party-content">hướng dẫn nội dung</a> trước khi đề xuất một thay đổi thêm một liên kết bên thứ ba nữa.</p>"""
+
+[thirdparty_message_vendor]
+other = """Các mục trên trang này đề cập đến các nhà cung cấp bên ngoài Kubernetes. Tác giả dự án Kubernetes không chịu trách nhiệm đối với những sản phẩm hoặc dự án của bên thứ ba đó. Để thêm một nhà cung cấp, sản phẩm hoặc dự án vào danh sách này, đọc <a href="/docs/contribute/style/content-guide/#third-party-content">hướng dẫn nội dung</a> trước khi gửi thay đổi. <a href="#third-party-content-disclaimer">Thông tin thêm.</a>"""
+
 
 [ui_search_placeholder]
-other = "Tìm kiếm"
+other = "Tìm kiếm trên trang này"
 
 [version_check_mustbe]
-other = "Server Kubernetes của bạn phải ở phiên bản "
+other = "Máy chủ Kubernetes của bạn phải là phiên bản "
 
 [version_check_mustbeorlater]
-other = "Server Kubernetes của bạn ở phiên bản mới hơn hoặc tại phiên bản "
+other = "Máy chủ Kubernetes của bạn phải là phiên bản từ hoặc sau phiên bản "
 
 [version_check_tocheck]
 other = "Để kiểm tra phiên bản, nhập "
 
+[version_menu]
+other = "Phiên bản"
+
 [warning]
 other = "Cảnh báo:"
 
 [whatsnext_heading]
-other = "Có gì tiếp theo"
-
+other = "Tiếp theo là gì"
```

I don't know Vietnamese, but just a suggestion to include a new empty line at the end of a file. This keeps the `red negative symbol` away and is considered a good practice - learned from dims :)
```suggestion
other = "Tiếp theo là gì"

```